meetpy/meetups/urls.py

```diff
@@ -6,7 +6,10 @@
 
 urlpatterns = [
     url(r'^spotkania/$', views.MeetupsListView.as_view(), name='meetup_list'),
-    url(r'^(?P<number>\d+)/$', views.MeetupDetailView.as_view(), name='meetup_detail'),
+    url(r'^(?P<meetup_type>[\w-]+)/(?P<number>\d+)/$',
```

Comment:
If we update to Django 2, can we switch `?P<name>` to `<str:meetup_type>`?

Reply:
Yes, path() sounds much better, although depends if we're going to merge this or the 2.1 branch first. Either way we can do it later on separate PR when both are merged.